nikola/__main__.py

```diff
@@ -238,16 +238,13 @@ def __init__(self, nikola, quiet=False):
         self.quiet = quiet
 
     def load_tasks(self, cmd, opt_values, pos_args):
+        DOIT_CONFIG = config.get('DOIT_CONFIG', {})
         if self.quiet:
-            DOIT_CONFIG = {
-                'verbosity': 0,
-                'reporter': 'zero',
-            }
+            DOIT_CONFIG['verbosity'] = 0
+            DOIT_CONFIG['reporter'] = 'zero'
         else:
-            DOIT_CONFIG = {
-                'reporter': ExecutedOnlyReporter,
-                'outfile': sys.stderr,
-            }
+            DOIT_CONFIG['reporter'] = ExecutedOnlyReporter
```

Comment:
Wouldn’t it be nicer to let users’ `DOIT_CONFIG` override this?  Some people might want to override our opinionated reporter choice.

```py
if self.quiet:
    DOIT_CONFIG = {…}  # from original code
else:
    DOIT_CONFIG = {…}

DOIT_CONFIG.update(config.get('DOIT_CONFIG', {}))
```

We also need to [pop `DOIT_CONFIG` out of the config dict](https://github.com/getnikola/nikola/blob/master/nikola/nikola.py#L285) in `nikola.py` (we don’t need to save it anywhere though) so `self.configured` works correctly.